package.json

```diff
@@ -210,13 +210,10 @@
             {
               "enum": [
                 "${workspaceFolder}/${workspaceFolderBasename}.jar",
-                "askUser",
                 ""
               ],
               "enumDescriptions": [
-                "%configuration.java.project.exportJar.targetPath.workspaceFolder%",
-                "%configuration.java.project.exportJar.targetPath.askUser%",
-                "%configuration.java.project.exportJar.targetPath.askUser%"
+                "%configuration.java.project.exportJar.targetPath.workspaceFolder%"
```

Comment:
Add a description for `""` as well?

Reply:
OK, added the description in https://github.com/microsoft/vscode-java-dependency/pull/575/commits/f1a42c2b5be3f895ab547cc9d24b8e206f220c5b